src/main.js

```diff
@@ -49,8 +49,13 @@ module.exports.apifyGoogleAuth = async ({ scope, tokensStore, credentials, googl
     const authorizeUrl = oAuth2Client.generateAuthUrl({
         access_type: 'offline',
         scope: `https://www.googleapis.com/auth/${scope}`,
+        // Always prompt for user consent otherwise, there is a chance to not obtain refresh_token
+        // https://stackoverflow.com/a/10857806
+        prompt: 'consent'
     });
-    let code;
+
+    const codeHolder = { code: null }; // To be able to access code as reference from any scope
```

Comment:
What is the scope we want to access this? We are sending it somewhere?

Reply:
The change in `code` was not detected in the `setInterval` body (I think due to different closure).